src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -162,6 +162,16 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
      */
     public selectionMeshPickedColor: Color3 = new Color3(0.3, 0.3, 1.0);
 
+    /**
+     * To be optionaly changed by user to define custom selection logic (after ray selection)
+     */
+    public meshSelectionPredicate: (mesh: AbstractMesh) => boolean;

```

Comment:
Please make sure a default value is set, as it seems that everything will fail if those two predicates are not initialized.

`() => true` would be great :-)

Reply:
I think I will go ahead and remove it because for my purposes the `meshSelectionPredicate` isn't needed and perhaps a little redundant!

Though I should point out the way it's written now it won't fail because in an `&&` conditional chain if the first value evaluates the to false the evaluation will short circuit.

Updating to remove!

Reply:
sorry, didn't see the first `this.meshSelectionPredicate && `